src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml

```
@@ -505,5 +509,8 @@ services:
             - '@akeneo_batch.job_repository'
             -
                 - '@pim_connector.step.delete_attributes'
+            - true
```

```suggestion
            - false
```
Not sure the delete_attributes job should be stoppable now

---

Agree with you ! If we allow stopping this job when a guy will stop the job the attribute will still be blacklisted

---

If we stop the job, the clean removed attribute job won't be triggered? Or we need to implement logic to launch the cleaning job when stopping